code/__HELPERS/icons.dm

```diff
@@ -1306,7 +1306,7 @@ GLOBAL_LIST_INIT(freon_color_matrix, list("#2E5E69", "#60A2A8", "#A1AFB1", rgb(0
 		if (isnull(icon_state))
 			icon_state = thing.icon_state
 
-			if (isnull(icon_state) || thing.flags_1 & HTML_USE_INITAL_ICON_1)
+			if (isnull(icon_state) || thing?.flags_1 & HTML_USE_INITAL_ICON_1)
```

Comment:
Shouldn't this still runtime because `thing` is directly accessed in the line above? (Unless the issue isn't resolved by `?`)

Reply:
The issue itself has to do with flags_1 being accessed. Which doesn't exist on non-atoms (byond moment being applied because people are calling this with mutable appearances as var1 in the code)